src/mscorlib/src/System/Globalization/CultureData.Unix.cs

```diff
@@ -14,6 +14,20 @@ namespace System.Globalization
 {
     internal partial class CultureData
     {
+        private static bool s_invariantGlobalizationMode = InitGlobalizationInvariantMode();
```

Comment:
Since corelib is crossgen'd, I assume the normal tricks around static readonly bools becoming JIT-time consts wouldn't apply, right?